### PR TITLE
Create 290-hardware-wallet-light

### PR DESCRIPTION
**Pull request by guylouis**
initial description of the swarm effort to launch our hardwallet light (uxr, ux, client integration with goeth and clojure, sourcing)

### CODE REVIEW

File: ideas/290-hardware-wallet-light [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+id: 290-hardware-wallet-light
+title: Hardware Wallet Light
+status: Active
+created: 2018-08-24
+category: core
+lead-contributor: bitgamma
```
Comment (oskarth) [inline ideas/290-hardware-wallet-light]: Isn't this you @guylouis?

File: ideas/290-hardware-wallet-light [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+Security:
+- a security audit of the javacard code has been delegated to an external security audit company
+
+## Dependency
+
+https://github.com/status-im/ideas/pull/292 is a dependency (whisper key decoupling)
+
+## Success Metrics
+
+### Phase 1
```
Comment (oskarth) [inline ideas/290-hardware-wallet-light]: Would be useful to have more fine granular 2 week iterations

File: ideas/290-hardware-wallet-light [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+   - printing of the card 
+   - realization of a custom packaging + tamper proof seal
+   - printing of our quick start guide 
+
+Security:
+- a security audit of the javacard code has been delegated to an external security audit company
+
+## Dependency
+
+https://github.com/status-im/ideas/pull/292 is a dependency (whisper key decoupling)
```
Comment (oskarth) [inline ideas/290-hardware-wallet-light]: clarification: dependency for launch, not for development

File: ideas/290-hardware-wallet-light [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+   - (if possible, not fully mandatory) supporting EC end point mutliplication  
+   - contactless (nfc)
+   - compatible with our app implementation
+- we are selecting a supplier or a set of suppliers able to provide to us:
+   - printing of the card 
+   - realization of a custom packaging + tamper proof seal
+   - printing of our quick start guide 
+
+Security:
+- a security audit of the javacard code has been delegated to an external security audit company
```
Comment (oskarth) [inline ideas/290-hardware-wallet-light]: rough timeline and impact here would be useful

File: ideas/290-hardware-wallet-light [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+## Dependency
+
+https://github.com/status-im/ideas/pull/292 is a dependency (whisper key decoupling)
+
+## Success Metrics
```
Comment (oskarth) [inline ideas/290-hardware-wallet-light]: Success metrics missing, some usage / number of cards etc we can use here?

File: ideas/290-hardware-wallet-light [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+lead-contributor: bitgamma
+contributors:
+    bitgamma
+    gravityblast
+    goranjovic
+    dmitryn
+    patrick
+    denis-sharpyn
+    guylouis
+
```
Comment (denis-sharypin) [inline ideas/290-hardware-wallet-light]: I would suggest adding @obi2020 as a content strategist. We need to explain a lot to users.
Also, we have a marketing website about a wallet. It has to be updated as well if anything has changed https://hardwallet.status.im/

File: ideas/290-hardware-wallet-light [hunk 7]
```diff
@@ -0,0 +1,123 @@
+---
+id: 290-hardware-wallet-light
+title: Hardware Wallet Light
+status: Active
+created: 2018-08-24
+category: core
+lead-contributor: bitgamma
+contributors:
+    bitgamma
+    gravityblast
```
Comment (pilu) [inline ideas/290-hardware-wallet-light]: if you want you can put my github username, @pilu

File: ideas/290-hardware-wallet-light [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+- UXR: @patrick
+- UX: @denis-sharpyn
+- PM:@guylouis
+
+## Product Overview
+
+The product is a HD hardware wallet in the form of a credit card size javacard with contactless capability (nfc), and is natively usable with our android clent.
+
+It will provide extended security and convenience to our users, providing:
+- ability to create or import an existing Status account or any BIP-32 wallet onto the javacard
```
Comment (pilu) [inline ideas/290-hardware-wallet-light]: for the first step can we only create new accounts? I think we are still not sure about importing existing accounts unless we ask the mnemonic.

**Comment by oskarth** [comment]
Ping on above @guylouis

**Pull request merged**